packages/plugins/rubygem-smart_proxy_omaha/rubygem-smart_proxy_omaha.spec

```
@@ -76,39 +96,32 @@ mkdir -p %{buildroot}%{foreman_proxy_settingsd_dir}
 mv %{buildroot}%{gem_instdir}/settings.d/omaha.yml.example \
    %{buildroot}%{foreman_proxy_settingsd_dir}/omaha.yml
 
-# crontab
-mkdir -p %{buildroot}%{_sysconfdir}/cron.d/
-mv %{buildroot}%{gem_instdir}/extra/foreman-proxy-omaha-sync.cron \
-   %{buildroot}%{_sysconfdir}/cron.d/%{name}
-
-# content directory
-mkdir -p %{buildroot}%{content_dir}
-
 %files
 %dir %{gem_instdir}
-%config(noreplace) %attr(0640, root, %{proxy_user}) %{foreman_proxy_settingsd_dir}/omaha.yml
-%attr(-,%{proxy_user},%{proxy_user}) %{content_dir}
-%doc %{gem_instdir}/LICENSE
+%{_bindir}/smart-proxy-omaha-sync
```

Something I am now wondering about: this will be in `/opt/tfm/usr/bin`. That's probably ok in most cases but this script also does some path loading manually so probably good to verify this

---

Thinking more about this: the cronjob actually calls his file so it should be:
```suggestion
%{_root_bindir}/smart-proxy-omaha-sync
```

This would also need a change to the `mv` command. The alternative (which may be safer) is to change the cronjob file to call `scl enable tfm smart-proxy-omaha-sync`